### PR TITLE
Fix candidate contact events

### PR DESCRIPTION
**Pull request by J-Dummermuth**

### CODE REVIEW

File: alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts [hunk 1]
```diff
@@ -118,10 +118,10 @@ export class CandidateSearchLoggingEffects {
   logEmailRav$ = this.logIdByAction<SelectRavEmailAction>(SELECT_RAV_EMAIL, CDEvent.RAV_EMAIL_CLICKED);
 
   @Effect({ dispatch: false })
-  logContactCandidateDialogOpened$ = this.logIdByAction<CandidateContactedAction>(CANDIDATE_CONTACTED, CDEvent.CONTACT_CANDIDATE_DIALOG_OPENED);
+  logContactCandidateDialogOpened$ = this.logIdByAction<ContactCandidateDialogSubmittedAction>(CONTACT_CANDIDATE_DIALOG_OPENED, CDEvent.CONTACT_CANDIDATE_DIALOG_OPENED);
```
Comment (birom) [inline alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts]: Please use the here ``ContactCandidateDialogOpenedAction``:
```
logContactCandidateDialogOpened$ = this.logIdByAction<ContactCandidateDialogOpenedAction>(CONTACT_CANDIDATE_DIALOG_OPENED, CDEvent.CONTACT_CANDIDATE_DIALOG_OPENED);
```

File: alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts [hunk 2]
```diff
@@ -118,10 +118,10 @@ export class CandidateSearchLoggingEffects {
...
 
   @Effect({ dispatch: false })
-  logCandidateContacted$ = this.logIdByAction<ContactCandidateDialogOpenedAction>(CONTACT_CANDIDATE_DIALOG_OPENED, CDEvent.CANDIDATE_CONTACTED);
+  logContactCandidateDialogSubmitted$ = this.logIdByAction<ContactCandidateDialogOpenedAction>(CONTACT_CANDIDATE_DIALOG_SUBMITTED, CDEvent.CONTACT_CANDIDATE_DIALOG_SUBMITTED);
```
Comment (birom) [inline alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts]: Please use here the ``ContactCandidateDialogSubmittedAction``:
```
logContactCandidateDialogSubmitted$ = this.logIdByAction<ContactCandidateDialogSubmittedAction>(CONTACT_CANDIDATE_DIALOG_SUBMITTED, CDEvent.CONTACT_CANDIDATE_DIALOG_SUBMITTED);
```

File: alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts (hunk 1 above)
Comment (99999989) [inline alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts]: The type parameter is wrong. Please change to `ContactCandidateDialogOpenedAction`.

File: alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts (hunk 2 above)
Comment (99999989) [inline alv-portal-ui/src/app/candidate/candidate-search/state-management/effects/candidate-search-logging-effects.ts]: Same here: The type parameter is wrong. Please change to `ContactCandidateDialogSubmittedAction `. thx

**Comment by J-Dummermuth** [comment]
Should be correct now

**Pull request merged**